bin/cron/activity-monitor

```diff
@@ -205,7 +205,7 @@ def main
         level_url = "http://studio.code.org/s/#{sl.script.id}/level/#{sl.id}"
       end
       log_string += %[ <a href="#{browserstack_url(i[:browser], level_url) || level_url}">✈ Repro It</a>]
-      HipChat.notify('ActivityMonitor', log_string, color: 'yellow', notify: 1)
+      HipChat.message 'ActivityMonitor', log_string, color: 'yellow', notify: 1
```

Comment:
Note that there is no reason to call `notify` if we are already passing in the `notify` parameter.

My hope is that the HB alert is caused by the `notify` value parameter being true (as a result of the merge on L47 in the method definition) rather than 1.